### DISCUSSION TITLE
fe_sendauth: no password supplied when adding node

### DISCUSSION THREAD

**Discussion by anthonator** [citusdata/citus#7038] [Q&A]
We currently have a multi-node setup with a coordinator node and one worker node. When we run `citus_add_node` we receive the the following error:

```
fe_sendauth: no password supplied
```

Here are the logs from the coordinator node:

```
connection to the remote node ***:5432 failed with the following error: fe_sendauth: no password supplied
```

The worker node has no relevant logs.

I'm not sure if it's having issues connecting to itself or the worker node. Since I'm not seeing anything in the worker node logs I'm leaning towards it having an issue communicating with itself.

We do have `citus.local_hostname` and `citus.node_conninfo` set on both nodes as well.

**Comment by playing-sudo** [comment]
You can edit pg_ Hba.conf, allowing coordination nodes and work nodes to trust each other
first ip in red rectangle is Coordinator。
for example:
![image](https://github.com/citusdata/citus/assets/56945569/162b48d6-2c1f-4d99-a879-61bb0478b353)

**Comment by hanefi** [comment]
IIRC, PostgreSQL servers does not log failed connection attempts. You can either update `pg_hba.conf` on all nodes or create .pgpass files or find another method that would allow connections from coordinator to the worker node.
